pkg/api/controllers/swagger.yaml

```diff
@@ -200,8 +224,20 @@ paths:
         "204":
           description: Empty response
         "400":
-          description: ""
-          content: {}
+          description: "Bad Request"
+          content:
+            application/json:
+              schema:
+                type: object
+                required:
+                  - error_code
+                properties:
+                  error_code:
```

Comment:
You could use the schema defined for the error_code property.

Reply:
I didn't use it, because I wanted to display an example making sense in the browser for the users, and not just `INTERNAL` `internal error`.